java/src/jmri/jmrit/audio/JoalAudioSource.java

```diff
@@ -265,28 +269,28 @@ public int numProcessedBuffers() {
     @Override
     public int numQueuedBuffers() {
         // First check we've been initialised
-        if (!_initialised) {
-            return (0);
+        if (!_initialised || !audioAlive()) {
+            return 0;
         }
 
-        int[] num_processed = new int[1];
-        // How many processed buffers are there?
-        al.alGetSourcei(_source[0], AL.AL_BUFFERS_QUEUED, num_processed, 0);
+        int[] num_queued = new int[1]; // klk  war num_processed
```

Comment:
What does this comment mean?

Reply:
Forgot to remove this. It appeared to me to be a copy-paste issue.